test/integration/visitor_can_add_item_to_cart_and_view_cart_test.rb

```diff
@@ -1,14 +1,77 @@
 require 'test_helper'
 
 class VisitorCanAddItemToCartAndViewCartTest < ActionDispatch::IntegrationTest
-  test 'visitor can add item to cart' do
-    skip
+  test 'visitor can add item to cart from index' do
+    Item.create(name: "gnar possum",
+                description: "a snowboard for shredding gnar pow",
+                price: 1000)
+
+    visit items_path
+    assert page.has_content?("Cart: 0")
+    click_link "Add To Cart"
+
+    assert page.has_content?("You now have 1 gnar possum in your cart.")
+    assert page.has_content?("Cart: 1")
+
+    click_link "Add To Cart"
+    assert page.has_content?("You now have 2 gnar possums in your cart.")
+    assert page.has_content?("Cart: 2")
+
+  end
+
+  test 'visitor can add item to cart from show page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.